### PR TITLE
Dan/etransfer

### PR DESCRIPTION
**Pull request by yhupaul**
I didn't make the select for From and To. the snackbar works and the creating transaction works.  I tried to complete what we talk about, today. Please, check and let me know if I missed something.

### CODE REVIEW

File: src/components/Transfer/Transaction/TxHistory.js [hunk 1]
```diff
@@ -20,55 +18,47 @@ const initialTx = {
...
     <Container component="main">
-            <Typography variant="h5" gutterBottom component="div" mt={2}>
-                Transaction History
+      <PageTitle title="Transaction History"/>
+      {txs.length !== 0 ?
+      <DynamicTable form={UserTxData} data={txs}/>
+                :
+                <Typography variant="h5" gutterBottom component="div" mt={2}>
+                No data
             </Typography>
```
Comment (smilecana) [inline src/components/Transfer/Transaction/TxHistory.js:59]: Add indentation

File: src/components/Transfer/Transaction/TxHistory.js [hunk 2]
```diff
@@ -20,55 +18,47 @@ const initialTx = {
...
+  }, [props.data._id])
+
+  //${props.data._id}
 
   const getData = () => {
-    axios.get(`http://localhost:5003/api/tx/${props.data._id}`)
+    axios.get(getApiRoute(API_USER_TX, {'userAccountId': props.data._id}))
         .then((res) => {
-          setTxs(res.data.userAccount.transactions);     
+          console.log(res);
```
Comment (smilecana) [inline src/components/Transfer/Transaction/TxHistory.js:34]: remove console.log

**Review by smilecana — COMMENTED** [review]
good

**Review by smilecana — APPROVED** [review]

File: backend/controllers/transferController.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        }
+    } catch (e) {
+        res.status(500).json({message: 'Internal Server Error'})
+    }
+}
+
+const createTransfer = async (req, res) => {
+    try {
+        const etResult = await new Transfer({...req.body}).save()
+        console.log(etResult)
```
[the remark this answers is not in the document]
Comment (Aleumkim) [reply backend/controllers/transferController.js:22]: I also agree that remove console log

File: src/components/Transfer/Transaction/TxHistory.js (hunk 1 above)
Comment (Aleumkim) [reply src/components/Transfer/Transaction/TxHistory.js]: organize indentation plz

**Review by Aleumkim — APPROVED** [review]

**Pull request merged**